components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/cache/EntitlementEngineCache.java

```diff
@@ -46,6 +46,7 @@
     private static final EntitlementEngineCache instance = new EntitlementEngineCache();
     private static CacheBuilder<Integer, EntitlementEngine> cacheBuilder;
     private static Log log = LogFactory.getLog(EntitlementEngineCache.class);
+    private static final long DEFAULT_ENTITLEMENT_ENGINE_CACHING_INTERVAL = 900;
```

Comment:
This fix is done to make the `EntitlementEngine` cache timeout configurable through a property in a `Entitlement.properties` file. The default value is used if the timeout is not configured in the property file, otherwise the value read from the property file will be used.